WcaOnRails/app/webpacker/components/Translators/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { Fragment, useMemo } from 'react';
+import { Container, Grid, Header } from 'semantic-ui-react';
+import _ from 'lodash';
+import useLoadedData from '../../lib/hooks/useLoadedData';
+import { apiV0Urls } from '../../lib/requests/routes.js.erb';
+import { groupTypes } from '../../lib/wca-data.js.erb';
+import Loading from '../Requests/Loading';
+import Errored from '../Requests/Errored';
+import I18n from '../../lib/i18n';
+import UserBadge from '../UserBadge';
+
+export default function Translators() {
+  const {
+    data: translators, loading, error,
+  } = useLoadedData(apiV0Urls.userRoles.listOfGroupType(groupTypes.translators));
+  const groupedTranslators = useMemo(
+    () => _.groupBy((translators || []), (role) => role.group.id),
+    [translators],
+  );
+
+  if (loading) return <Loading />;
+  if (error) return <Errored />;
+
+  return (
+    <Container fluid>
+      <Header as="h2">{I18n.t('page.translators.title')}</Header>
+      {Object.keys(groupedTranslators).map((groupId) => (
+        <Fragment key={groupId}>
+          <Header as="h3">{groupedTranslators[groupId][0].group.name}</Header>
+          <Grid padded>
+            {groupedTranslators[groupId].map((role) => (
+              <Grid.Column
+                key={role.id}
+                style={{ width: 'fit-content' }}
```

Comment:
Please try to avoid inline CSS. There's a 95% chance that SemUI already has some properties on its components that can do the same or similar thing you're trying to achieve.

Reply:
Ya I understand. I tried a lot, but couldn't find any alternative in SemUI.

Reply:
What are you trying to achieve with this custom CSS?

Reply:
Attaching the screenshot with the current CSS:
<img width="1069" alt="Screenshot 2024-02-29 at 06 24 18" src="https://github.com/thewca/worldcubeassociation.org/assets/9512698/dd52d936-fd77-4df9-9897-fe7d1e9035b4">

If the CSS is not there, UI will be:
<img width="878" alt="Screenshot 2024-02-29 at 06 26 50" src="https://github.com/thewca/worldcubeassociation.org/assets/9512698/3252e8b2-088a-403d-a6fc-874ac8451c27">

And another thing that I just now tried was removing `Grid.Column` and it worked. I didn't try that earlier because I had an assumption that everything inside Grid should have Grid.Column. Anyway, I've pushed the code without Grid.Column, which should give expected UI without extra inline CSS.

Reply:
The clue here is `GridRow`. Your assumption that everything needs `Grid.Column` is not wrong, but it's incomplete. You also need `Grid.Row` to wrap columns. But anyways, it's fine to keep it in the current way as a shorthand.